dream-mirage.opam

```diff
@@ -60,6 +60,7 @@ depends: [
   "tls-mirage"
   "lwt_ppx" {>= "1.2.2"}
   "letsencrypt" {>= "0.3.0"}
+  "emile" {>= "1.1"}
```

Comment:
It's a bit unfortunate 😕 . Currently, only `paf-le` (which is another package to handle Let's encrypt certificates) depends on `emile`. Not sure which direction you want to take, but if you don't use `paf-le` and Let's encrypt certificate, you should not have such dependency.

Reply:
Thanks! I'll note that for a separate issue. I want to get these commits over into Dream in about the same form that they are already in, and then edit them further.